src/hooks/useMapState.ts

```diff
@@ -6,82 +6,81 @@ import { useCallback, useState } from "react";
  *
  * @param initialValue Initial value of the map
  */
-function useMapState(
-  initialValue: any
-): [
-  any,
+function useMapState<T>(initialValue: T): [
+  T,
   {
-    set: (key: any, value: any) => void;
-    has: (key: any) => boolean;
-    setMultiple: (...keys: any[]) => void;
-    remove: (key: any) => void;
-    removeMultiple: (...keys: any[]) => void;
+    has: (key: keyof T) => boolean;
```

Comment:
@qqpann Nice. Let's alias this into a type at the top of this file and use it here for good readibility.

Reply:
I changed it. With this change, you have to specify the second type when explicitly specifying genetics